docs/cypress.config.js

```diff
@@ -1,6 +1,7 @@
 const { defineConfig } = require("cypress");
 
 module.exports = defineConfig({
+  projectId: 'u5oh2w',
```

Comment:
Please confirm why do we need to specify `projectId` here?

Reply:
reverted this changes in the latest commit